public/battle.js

```diff
@@ -0,0 +1,21 @@
+async function startBattle() {
+    const response =  fetch('/api/import-songs/songs', {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      })
+    //   .then(response =>{
+    //     console.log(response)
+
+      
+      if (response.ok) {
+        //   console.log("It worked")        
+          await redirect("/home")
```

Comment:
This code fires when the begin battle is clicked. It doesn't reload the new random roll though? Need to think about front logic..